manifests/cf-manifest/spec/manifest/rds_broker_spec.rb

```diff
@@ -178,16 +177,23 @@
           "xlarge-9.5",
           "xlarge-ha-unencrypted-9.5",
           "xlarge-ha-9.5",
+
```

Comment:
This empty line appears to be making Travis fail.

Reply:
Sorry, removed.

Reply:
👍